cmake/install.cmake

```diff
@@ -100,9 +101,19 @@ configure_file(protobuf-config-version.cmake.in
 configure_file(protobuf-module.cmake.in
   protobuf-module.cmake @ONLY)
 
+# Allows the build directory to be used as a find directory.
+export(EXPORT protobuf-targets
+  NAMESPACE protobuf::)
```

Comment:
Oops... [CMP0024](https://cmake.org/cmake/help/latest/policy/CMP0024.html) added in CMake 3.0 and we need and we promised to support CMake version 2.8.12 which does not have export(EXPORT..., exportr(TARGETS.. it is only available (see [there](https://cmake.org/cmake/help/v2.8.12/cmake.html#command:export)).